includes/sqp/1-4k-cf-scoring-sqp1.md

```diff
@@ -1,102 +1,101 @@
     | [{{ radarr['cf']['criterion-collection']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#criterion-collection) | {{ radarr['cf']['criterion-collection']['trash_score'] }} | {{ radarr['cf']['criterion-collection']['trash_id'] }} |
-    | [{{ radarr['cf']['masters-of-cinema']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#masters-of-cinema)       | {{ radarr['cf']['masters-of-cinema']['trash_score'] }}    | {{ radarr['cf']['masters-of-cinema']['trash_id'] }}    |
-    | [{{ radarr['cf']['vinegar-syndrome']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#vinegar-syndrome)         | {{ radarr['cf']['vinegar-syndrome']['trash_score'] }}     | {{ radarr['cf']['vinegar-syndrome']['trash_id'] }}     |
-    | [{{ radarr['cf']['special-edition']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#special-edition)           | {{ radarr['cf']['special-edition']['trash_score'] }}      | {{ radarr['cf']['special-edition']['trash_id'] }}      |
-    | [{{ radarr['cf']['imax']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#imax)                                 | {{ radarr['cf']['imax']['trash_score'] }}                 | {{ radarr['cf']['imax']['trash_id'] }}                 |
-    | [{{ radarr['cf']['imax-enhanced']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#imax-enhanced)               | {{ radarr['cf']['imax-enhanced']['trash_score'] }}        | {{ radarr['cf']['imax-enhanced']['trash_id'] }}        |
+    | [{{ radarr['cf']['masters-of-cinema']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#masters-of-cinema)       |  {{ radarr['cf']['masters-of-cinema']['trash_score'] }}   | {{ radarr['cf']['masters-of-cinema']['trash_id'] }}    |
+    | [{{ radarr['cf']['vinegar-syndrome']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#vinegar-syndrome)         |   {{ radarr['cf']['vinegar-syndrome']['trash_score'] }}   | {{ radarr['cf']['vinegar-syndrome']['trash_id'] }}     |
+    | [{{ radarr['cf']['special-edition']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#special-edition)           |   {{ radarr['cf']['special-edition']['trash_score'] }}    | {{ radarr['cf']['special-edition']['trash_id'] }}      |
 
-    !!! tip "{{ radarr['cf']['imax-enhanced']['name'] }}"
-        {{ radarr['cf']['imax-enhanced']['name'] }}: Get More Picture Instead of Black Bars.
+    !!! info
+        - If you prefer 1080p/2160p WEBDL with IMAX-E then add [{{ radarr['cf']['imax-enhanced']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#imax-enhanced) with the default scores.
+        - The reason why we don't add [{{ radarr['cf']['imax']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#imax) is because BHDStudio doesn't add it to their release name, their motto if the source has imax then it will be used.
```

Comment:
```suggestion
        - The reason why we don't add [{{ radarr['cf']['imax']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#imax) is because BHDStudio doesn't add it to their release name. Their motto is: If the source has IMAX then the encode will have it as well.
```